test/ui/ResultActionsMenuTest.ts

```diff
@@ -0,0 +1,44 @@
+import * as Mock from '../MockEnvironment';
+import { IQueryResult } from '../../src/rest/QueryResult';
+import { FakeResults } from '../Fake';
+import { ResultActionsMenu } from '../../src/ui/ResultActions/ResultActionsMenu';
+import { $$ } from '../../src/utils/Dom';
+
+export function ResultActionsMenuTest() {
+  describe('ResultActionsMenu', () => {
+    let resultActions: ResultActionsMenu;
+    let test: Mock.IBasicComponentSetup<ResultActionsMenu>;
+    let fakeResult: IQueryResult;
+
+    beforeEach(() => {
+      fakeResult = FakeResults.createFakeResult();
+      test = Mock.advancedResultComponentSetup<ResultActionsMenu>(ResultActionsMenu, fakeResult, undefined);
+      spyOn(test.cmp, 'show');
```

Comment:
Why do you need to spy on the method, if you are not testing that it gets called, but simply that the component has the correct class ?